src/drizzleOptions.js

```diff
@@ -55,7 +55,7 @@ const options = {
   contracts: contracts,
   events: {
     WizardPresale: ["ApprovalForAll"],
-    KittyCore: ["ApprovalForAll"],
+    KittyCore: ["Approval"],
     ManaBank: ["GetMana", "BurnMana"]
```

Comment:
This is probably the problem you mentioned about drizzle. Unfortunately, `KittyCore` does not have the standard ERC721 contract. It does not have `setApprovalForAll` method, and `ApprovalForAll` event. We need to ask the user to approve each kitty, oh well. @lsankar4033

Reply:
damn, i forgot about that